src/main/java/tree/ImpTargetTreeNodeComparator.kt

```diff
@@ -0,0 +1,9 @@
+package tree
+
+class ImpTargetTreeNodeComparator : Comparator<ImpTargetTreeNode> {
+    override fun compare(o1: ImpTargetTreeNode?, o2: ImpTargetTreeNode?): Int {
```

Comment:
Params with more signification.
```suggestion
    override fun compare(node1: ImpTargetTreeNode?, node2: ImpTargetTreeNode?): Int {
```